corehq/apps/app_manager/_design/views/saved_app/map.js

```diff
@@ -24,6 +24,7 @@ function(doc){
             vellum_case_management: !!doc.vellum_case_management,
             target_commcare_flavor: doc.target_commcare_flavor,
             family_id: doc.family_id,
+            profile: doc.profile,
```

Comment:
If this is changed for the limited release, I presume this will actually make the change to the couch view? I need to quickly find out what will happen when the release expires.